src/pages/MainPage.vue

```diff
@@ -373,7 +375,9 @@ export default {
     roads: {
       handler: function (newRoads, oldRoads) {
         this.justLoaded = false;
-        if (this.activeRoad !== '') {
+        this.cookiesAllowed = true;
+        this.$refs.authcomponent.allowCookies();
+        if (this.activeRoad != '') {
```

Comment:
is this supposed to be a single `=`?

Reply:
No, I think this happened because this branch started before the linter.  Will fix when I have time to get to my computer.